notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #74.        np.save('./data/results/main.npy',results) 
</pre><p>It seems the numpy array is overwritten at each iteration step. I would suggest to move the line outside of the loop to avoid that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Thank you. The saving operation is outside the loop.</p>

Reply:
<p>closing conversation, resolved</p>